### PR TITLE
add firmware licence and revert the patch which break compatibility with upstream kernel

### PR DESCRIPTION
**Pull request by abhimanyuv1**
I think we should create new release once we merge this one?

### CODE REVIEW

File: LICENSE.imagination [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2015 Imagination Technologies Limited and/or its affiliated group companies
```
Comment (abhijit-mahajani-imgtec) [inline LICENSE.imagination]: Should this be in "firmware" folder, rather than here?
@chaitu-tk what is your opinion?

**Comment by abhimanyuv1** [comment]
i have move the licence to firmware directory for more clarity

**Review by tkc17 — COMMENTED** [review]
Overall looks ok to me. Can you please point me to the source of the license file?

**Comment by abhimanyuv1** [comment]
It is from the linux-firmware package from internal imgwork repo which accompanied the firmware. Please check the imgwork repo for the same.

**Review by tkc17 — APPROVED** [review]
Thanks Abhi.

**Comment by tkc17** [comment]
Hi Abhi,

I did not realize that this merge was directly in to REL, the procedure is first you pull it to DEV and then to REL. I will revert this, can you please raise a new PR in to DEV?

Sorry for the overlook.

**Pull request merged**